nrf_modem/doc/sockets.rst

```diff
@@ -10,326 +10,14 @@ Networking sockets
-     This status is returned before the DTLS handshake is complete.
-   * :c:macro:`NRF_SO_SEC_DTLS_CID_STATUS_DOWNLINK` - The DTLS connection ID is included only in DTLS records sent to the modem.
-   * :c:macro:`NRF_SO_SEC_DTLS_CID_STATUS_UPLINK` - The DTLS connection ID is included only in DTLS records sent from the modem.
-   * :c:macro:`NRF_SO_SEC_DTLS_CID_STATUS_BIDIRECTIONAL` - The DTLS connection ID is included in DTLS records sent to and from the modem.
-
-NRF_SO_SEC_DTLS_CONN_SAVE
-   Save DTLS connection.
-   This socket option can be used to pause a session that is not frequently used by the application.
-   Saving the session will free memory in the modem, so the memory can be used for other connections.
-   If the socket is closed, the saved DTLS data is cleaned and the connection with the server is lost.
-
-   This option is write-only.
-   This option require a DTLS v1.2 connection with renegotiation disabled.
-   The socket option is supported from modem firmware v1.3.x, where x is greater than or equal to 5, and v2.x.x.
-
-   Once the DTLS context is saved, the socket can't be used before the DTLS context is loaded with :c:macro:`NRF_SO_SEC_DTLS_CONN_LOAD`.
-
-   This option fails with nrf_errno ``NRF_EAGAIN`` if an error happened during serialization of the SSL context.
-   This can occur, for instance, when the modem cannot allocate enough memory or if the socket is busy sending or receiving data.
-   In this case, the SSL context is still present in the socket, so data sending is still possible.
-   The option fails with nrf_errno ``NRF_EINVAL`` if the socket option is not supported with the current configuration, for instance because the DTLS handshake is not completed,
-   the connection is not an DTLS v1.2 connection with renegotiation disabled, or the connection does not use an AEAD cipher suite (AES-CCM or AES-GCM).
-   The option fails with nrf_errno ``NRF_ENOMEM`` if the amount of saved connections exceeds four.
-
-NRF_SO_SEC_DTLS_CONN_LOAD
-   Load DTLS connection.
-   This option is write-only.
-   The socket option is supported from modem firmware v1.3.x, where x is greater than or equal to 5, and v2.x.x.
-
-   This option fails with nrf_errno ``NRF_EAGAIN`` if an error happened during deserialization of the SSL context.
-   This can occur, for instance, when the modem cannot allocate enough memory or the connection is not saved.
-   The option fails with nrf_errno ``NRF_EINVAL`` if the socket option is not supported with the current configuration.
-
-NRF_SO_SEC_CIPHERSUITE_USED
-   Get chosen TLS cipher suite.
-   This option is read-only.
-   The socket option is supported from modem firmware v2.x.x.
-
-.. include:: sockets/pdn.rst
-
-.. include:: sockets/tls_dtls.rst
-
-.. include:: sockets/rai.rst
+   sockets/socket_options_func
+   sockets/pdn
+   sockets/tls_dtls
+   sockets/rai
```

Comment:
I think I prefer this solution with sub-paging better than including/embedding the other rst files into sockets.rst. The "networking sockets" page looks clean and it is easy to see what topics the subpages address.